### PR TITLE
fix not unpack ref holder for list

### PR DESCRIPTION
**Pull request by wongoo**
**What this PR does**:
- fix not unpack ref holder for list

**Which issue(s) this PR fixes**:
ref later.

### CODE REVIEW

**Comment by wongoo** [comment]
this pr fix https://github.com/apache/dubbo-go-hessian2/issues/270

File: codec.go [hunk 1]
```diff
@@ -422,9 +415,21 @@ func SetSlice(dest reflect.Value, objects interface{}) error {
 	return nil
 }
 
+// unpackRefHolder unpack the ref holder when decoding slice finished.
```
Comment (AlexStocks) [inline codec.go:418]: unpacks

**Review by AlexStocks — APPROVED** [review]

**Review by zouyx — APPROVED** [review]

File: codec.go (hunk 1 above)
Comment (wongoo) [reply codec.go:418]: I think should use do not does.

**Comment by wongoo** [comment]
@fangyincheng  pls review ~

**Pull request merged**